abci/client/mocks/client.go

```diff
@@ -837,3 +837,18 @@ func (_m *Client) String() string {
 
 	return r0
 }
+
+type mockConstructorTestingTNewClient interface {
```

Comment:
I would like to know why this was necessary.

Reply:
This was generated automatically by mockery.